### ISSUE TITLE
Books stored in smartphone are new instances of the book type

### ISSUE THREAD

**Issue by AideeAmei** [CleverRaven/Cataclysm-DDA#77920]
### Describe the bug

When player reads books, the books lose charge equal to its ability to boost player's status effectively (chapters are read and player knows them so they're not new and interesting anymore). When a book is stored in a smartphone, the copy stored doesn't reflect this, as if it was a different book with different chapters.

### Attach save file

N/A

### Steps to reproduce

1. Store a book into a smartphone.
2. Read a chapter of the physical copy.
3. Check read menu, there's a difference between the physical copy and the smartphone version - the digital copy didn't lose a chapter.

### Expected behavior

Reading a chapter from the book reads the chapter from all its copies.

### Versions and configuration

OS: Windows 10 Pro 64-bit 22H2 19045.5131
Game Version: 0f2c155 (tiles)
build 2024-11-15-0512
Ingame Language: Enskt

### Additional context

Or is it like when playing games on different platforms is a different experience? :)

**Comment by ZhilkinSerg** [comment]
Yes, these physical and digital books are different items. Remaining chapters are not copied over when the book is scanned.

**Comment by PatrikLundell** [comment]
Does this mean you can keep reading different copies of the same book over and over as long as you change the particular copy between each read?

**Comment by ZhilkinSerg** [comment]
Yes, you can read different copies of the same book to spend chapters independently

![image](https://github.com/user-attachments/assets/e9f2ebe6-4c2e-4739-8fec-eac1b1273aa0)

**Comment by AideeAmei** [comment]
Basically finding a smartphone is like finding all books you found so far. As long as you keep the hard copies unread, it's an infinite source of fun.

**Comment by PatrikLundell** [comment]
Or you could just not abuse the deficiency (not saying it oughtn't to be fixed eventually).

**Comment by ZhilkinSerg** [comment]
No, it is not different from physical books whatsoever

**Comment by AideeAmei** [comment]
I'm not abusing it, on contrary, I reported it! 😆
